// react-router/dist/development/index-react-server-client.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});/**
 * react-router v7.16.0
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */
"use client";










var _chunkIBI7OMNBjs = require('./chunk-IBI7OMNB.js');














var _chunkSRID2YZ2js = require('./chunk-SRID2YZ2.js');























exports.BrowserRouter = _chunkIBI7OMNBjs.BrowserRouter; exports.Form = _chunkIBI7OMNBjs.Form; exports.HashRouter = _chunkIBI7OMNBjs.HashRouter; exports.Link = _chunkIBI7OMNBjs.Link; exports.Links = _chunkSRID2YZ2js.Links; exports.MemoryRouter = _chunkSRID2YZ2js.MemoryRouter; exports.Meta = _chunkSRID2YZ2js.Meta; exports.NavLink = _chunkIBI7OMNBjs.NavLink; exports.Navigate = _chunkSRID2YZ2js.Navigate; exports.Outlet = _chunkSRID2YZ2js.Outlet; exports.Route = _chunkSRID2YZ2js.Route; exports.Router = _chunkSRID2YZ2js.Router; exports.RouterProvider = _chunkSRID2YZ2js.RouterProvider; exports.Routes = _chunkSRID2YZ2js.Routes; exports.ScrollRestoration = _chunkIBI7OMNBjs.ScrollRestoration; exports.StaticRouter = _chunkIBI7OMNBjs.StaticRouter; exports.StaticRouterProvider = _chunkIBI7OMNBjs.StaticRouterProvider; exports.UNSAFE_AwaitContextProvider = _chunkSRID2YZ2js.AwaitContextProvider; exports.UNSAFE_WithComponentProps = _chunkSRID2YZ2js.WithComponentProps; exports.UNSAFE_WithErrorBoundaryProps = _chunkSRID2YZ2js.WithErrorBoundaryProps; exports.UNSAFE_WithHydrateFallbackProps = _chunkSRID2YZ2js.WithHydrateFallbackProps; exports.unstable_HistoryRouter = _chunkIBI7OMNBjs.HistoryRouter;
