// react-router/dist/production/index-react-server-client.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});/**
 * react-router v7.16.0
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */
"use client";










var _chunkSKEDDLRMjs = require('./chunk-SKEDDLRM.js');














var _chunkY7DNFQZPjs = require('./chunk-Y7DNFQZP.js');























exports.BrowserRouter = _chunkSKEDDLRMjs.BrowserRouter; exports.Form = _chunkSKEDDLRMjs.Form; exports.HashRouter = _chunkSKEDDLRMjs.HashRouter; exports.Link = _chunkSKEDDLRMjs.Link; exports.Links = _chunkY7DNFQZPjs.Links; exports.MemoryRouter = _chunkY7DNFQZPjs.MemoryRouter; exports.Meta = _chunkY7DNFQZPjs.Meta; exports.NavLink = _chunkSKEDDLRMjs.NavLink; exports.Navigate = _chunkY7DNFQZPjs.Navigate; exports.Outlet = _chunkY7DNFQZPjs.Outlet; exports.Route = _chunkY7DNFQZPjs.Route; exports.Router = _chunkY7DNFQZPjs.Router; exports.RouterProvider = _chunkY7DNFQZPjs.RouterProvider; exports.Routes = _chunkY7DNFQZPjs.Routes; exports.ScrollRestoration = _chunkSKEDDLRMjs.ScrollRestoration; exports.StaticRouter = _chunkSKEDDLRMjs.StaticRouter; exports.StaticRouterProvider = _chunkSKEDDLRMjs.StaticRouterProvider; exports.UNSAFE_AwaitContextProvider = _chunkY7DNFQZPjs.AwaitContextProvider; exports.UNSAFE_WithComponentProps = _chunkY7DNFQZPjs.WithComponentProps; exports.UNSAFE_WithErrorBoundaryProps = _chunkY7DNFQZPjs.WithErrorBoundaryProps; exports.UNSAFE_WithHydrateFallbackProps = _chunkY7DNFQZPjs.WithHydrateFallbackProps; exports.unstable_HistoryRouter = _chunkSKEDDLRMjs.HistoryRouter;
